release_deploy_timeline.md

```diff
@@ -1,5 +1,6 @@
 # deploy-longhaul-release timeline
 
+- 2024-08-08 - Updated dapr version to 1.14.0-rc.8
```

Comment:
Should this not be `1.14.0-rc.11`?

Reply:
No, this is for the 8th of August, when we moved to rc8. Once we move to rc11 I'll update the timeline file with the correct date.

Reply:
> No, this is for the 8th of August, when we moved to rc8. Once we move to rc11 I'll update the timeline file with the correct date.

Just a thought, is a downside of updating notes post-changes a cause for potential redeployment of resources?

Reply:
Ah no, this note is just for our own convenience, so we have a single place to look at when we're looking at the graphs on a timeline, and we don't have to go and look which PRs were merged when.